doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -253,7 +254,9 @@ Doc. ID: UG-SS
   - [19.3 Correlation ID HTTP header](#193-correlation-id-http-header)
   - [19.4 Validation errors](#194-validation-errors)
   - [19.5 Warning responses](#195-warning-responses)
-- [20 Migrating to Remote Database Host](#20-migrating-to-remote-database-host)
+- [20 Remote Database Host](#20-remote-database-host)
+-   [20.1 Migrating to Remote Database Host](#201-migrating-to-remote-database-host)
+-   [20.2 Passing additional parameters to psql](#202-passing-additional-parameters-to-psql)
```

Comment:
I think we should have this as a completely separate step, since you could technically do this for local databases as well, even if that use case is probably pretty rare.